n2y/plugins/linktopage.py

```diff
@@ -0,0 +1,58 @@
+import logging
+
+from n2y.blocks import LinkToPageBlock
+
+logger = logging.getLogger(__name__)
+
+
+class LinkToPage(LinkToPageBlock):
+    """
+    Replace page link with the content of the linked-to page.
+
+    """
+
+    def __init__(self, client, notion_data, page, get_children=True):
+
+        super().__init__(client, page, notion_data, get_children)
+
+        # The type of object link can be either "page_id" or "database_id"
+        self.link_type = self.notion_data["type"]
+        self.linked_page_id = self.notion_data[self.link_type]
+        self.linking_page_id = self.page.notion_id
+
+    def to_pandoc(self):
+        assert self.linked_page_id is not None
+
+        # TODO: Might be expanded to handle links to databases as well.
+        if self.link_type == "page_id":
+
+            try:
+                page = self.client.get_page_or_database(self.linked_page_id)
+
+                rich_text = self.client.wrap_notion_rich_text_array(
```

Comment:
@idolcemia can you explain the reasoning behind this line of code? I don't think it really makes sense and I'm surprised that the tests are passing.